src/pkg/token/options.go

```diff
@@ -76,3 +59,20 @@ func (o *Options) GetKey() (interface{}, error) {
 		return nil, fmt.Errorf(fmt.Sprintf("unsupported sign method, %s", o.SignMethod))
 	}
 }
+
+// DefaultTokenOptions ...
+func DefaultTokenOptions() *Options {
+	privateKeyFile := config.TokenPrivateKeyPath()
+	privateKey, err := ioutil.ReadFile(privateKeyFile)
+	if err != nil {
+		log.Errorf(fmt.Sprintf("failed to read private key %v", err))
```

Comment:
A general function, maybe an error should be returned, not logged

Reply:
the token handler can throw the real issue.